docs/docs/user-guide/use-cli-api.md

```diff
@@ -4,34 +4,44 @@ Typically, you'll work with CLI for Microsoft 365 in a command line. You'll eith
 
 ## Integrate CLI for Microsoft 365 in your app
 
-If you build apps in Node.js, you can integrate CLI for Microsoft 365 using its API. This API lets you call any of CLI's commands. The following example shows how you could call several CLI for Microsoft 365 commands in a Node.js app:
+If you build apps in Node.js, you can integrate CLI for Microsoft 365 using its API. This API lets you call any of the CLI's commands. We recommend to use the await operator to wait for the promise returned from the executeCommand function. The following examples show how you could call several CLI for Microsoft 365 commands in a Node.js app:
```

Comment:
I'd say, let's remove the part about using `await` it's up to the developer to decide what he/she likes to use. It's not wrong to use promise/then.